src/main/resources/META-INF/resources/webjars/service/prov/lib/stacked.js

```diff
@@ -134,11 +134,11 @@ define(['d3', 'jquery'], function (d3) {
             }
         }
 
-        function update(data, aggregateMode) {
+        function update({ data, aggregateMode, tooltip }) {
```

Comment:
Also need to update the call from `prov.js`: `d3Bar.update(data, aggregateMode);`